gamestonk_terminal/cryptocurrency/README.md

```diff
@@ -102,13 +110,11 @@ Gets and shows the order book for the given coin/currency.  Shows the cumulative
 
 ## candle  <a name="candle"></a>
 ````
-usage: candle [-i --interval] [-l --limit]
+usage: candle
```

Comment:
Was the removal of the interval and limit intentional?

Reply:
Yeah.  `select` now does that and candle just plots it.